app/models/manageiq/providers/configuration_manager.rb

```diff
@@ -1,11 +1,7 @@
 class ManageIQ::Providers::ConfigurationManager < ::ExtManagementSystem
-  require_nested :InventoryGroup
-  require_nested :InventoryRootGroup
-
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
-  has_many :inventory_groups,             :dependent => :destroy, :foreign_key => "ems_id", :inverse_of => :manager
```

Comment:
It was introduced for Ansible Tower with the thought that it would be reused in Foreman (but that hasn't been done yet).